MekHQ/src/mekhq/campaign/force/Lance.java

```diff
@@ -227,7 +227,8 @@ public int getWeightClass(Campaign c) {
 
     public boolean isEligible(Campaign c) {
         // ensure the lance is marked as a combat force
-        if (!c.getForce(forceId).isCombatForce()) {
+        final Force force = c.getForce(forceId);
+        if ((force == null) || !force.isCombatForce()) {
```

Comment:
Can we also annotate Campaign::getForce as having a nullable return? Would assist in preventing this in future.